CONTRIBUTING.md

```diff
@@ -26,6 +26,16 @@ Before working on an issue, there are a few things that you need to pay attentio
 2. To start working on it, ensure that you create a new branch from the `main` branch, then commit and push your changes **as soon as possible** no matter how small they are.
 3. Then **create a new pull request** while [marking it as a **draft pull request**](https://github.blog/2019-02-14-introducing-draft-pull-requests/) to signal the other contributors that [it's a work in progress](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request). This is necessary to signal the other contributors that there are work in progress for that particular issue.
 
+## FAQ
+
+### Why are you using English in your issues & PRs?
+
+There are several reasons we're using English while communicating in GitHub Issues & PRs:
+
+1. It's more natural for software engineers to communicate in English because it involves many technical terms in English. Trying to translate them into Bahasa Indonesia posing a risk of miscommunication, while keeping them in English requires us to do a lot of _italic_ formatting, according to [PUEBI](https://puebi.js.org/huruf/miring).
```

Comment:
```suggestion
1. It's more natural for software engineers to communicate in English because it involves many technical terms in English. Trying to translate them into Bahasa Indonesia posing a risk of miscommunication, while keeping them in English requires us to do a lot of italic formatting, according to [PUEBI](https://puebi.js.org/huruf/miring).
```

do we need it as _italic_?

Reply:
Yes, we do, to emphasize the word, it's a common usage in English.